cmdb/interface/rest_api/search_routes.py

```diff
@@ -84,14 +84,10 @@ def search_framework(request_user: UserModel):
     try:
         searcher = SearcherFramework(manager=object_manager)
         builder = SearchPipelineBuilder()
-        if request.method == 'GET' and resolve_object_references:
-            query: Pipeline = builder.build_resolve_reference_pipeline(query=Query(search_parameters),
-                                                                       active=only_active, user=request_user,
-                                                                       permission=AccessControlPermission.READ)
-        else:
-            query: Pipeline = builder.build(search_parameters, object_manager,
-                                            user=request_user,
-                                            permission=AccessControlPermission.READ, active_flag=only_active)
+
+        query: Pipeline = builder.build(search_parameters, object_manager,
+                                        user=request_user,
+                                        permission=AccessControlPermission.READ, active_flag=only_active)
```

Comment:
Wouldn't it be cleaner to set a resolution of the active flag before?
For me it is the same:
`active` -> `{"active": true}` in search_parameters